greedy_algorithms/digit_separation.cpp

```diff
@@ -0,0 +1,132 @@
+#include <algorithm>
```

Comment:
Please add a header comment following the format defined in the contributing.md file

Reply:
Done